examples/computer_vision/detr_coco_pytorch/data_utils.py

```diff
@@ -0,0 +1,52 @@
+import os
+import asyncio
+from io import BytesIO
+from tempfile import NamedTemporaryFile
+import requests
+from shutil import unpack_archive
+
+
+def download_file(url, output_dir):
+    local_filename = os.path.join(output_dir, url.split("/")[-1])
+    with requests.get(url, stream=True) as r:
+        r.raise_for_status()
+        with open(local_filename, "wb") as f:
+            for chunk in r.iter_content(chunk_size=8192):
+                # If you have chunk encoded response uncomment if
+                # and set chunk_size parameter to None.
```

Comment:
Would condense the two line explanation of comment into one just for readibility, and say uncomment "the line below"